docs/chainlink-keepers/overview.html

```diff
@@ -66,15 +66,28 @@
 
 ### {{network[0]}}
 
-* **Payment Premium (paymentPremiumPPB)** - <span id="show-{{network[1].id}}-paymentPremiumPPB"></span>% gas premium
+* **Payment Premium (paymentPremiumPPB)** - <span id="show-{{network[1]}}-paymentPremiumPPB"></span>% gas premium
     This premium is applied on top of the gas cost paid by the Keeper to perform your Upkeep
-* **Block Count Per Turn (blockCountPerTurn)** - <span id="show-{{network[1].id}}-blockCountPerTurn"></span>
+* **Block Count Per Turn (blockCountPerTurn)** - <span id="show-{{network[1]}}-blockCountPerTurn"></span>
     The number of blocks in each turn before the Upkeep is rotated to the next Keeper
-* **Check Gas Limit (checkGasLimit)** - <span id="show-{{network[1].id}}-checkGasLimit"></span>
+* **Check Gas Limit (checkGasLimit)** - <span id="show-{{network[1]}}-checkGasLimit"></span>
     The maximum amount of gas that can be used by your checkUpkeep for off-chain computation.
 * **Call Gas Limit (callGasLimit)**: <span>2,500,000</span> <!-- @TODO render this programatically. <span id="show-callGasLimit"></span> -->
     The maximum amount of gas that can be used by the client contract’s performUpkeep for the on-chain transaction. You can set an upper limit on your Upkeep during registration, but this number must not exceed the call gas limit on the KeeperRegistry.
-* **Gas Ceiling Multiplier (gasCeilingMultiplier)** - <span id="show-{{network[1].id}}-gasCeilingMultiplier"></span>
+* **Gas Ceiling Multiplier (gasCeilingMultiplier)** - <span id="show-{{network[1]}}-gasCeilingMultiplier"></span>
+    Establishes a ceiling for the maximum price based on the on-chain fast gas feed.
+
+### Binance Smart Chain
+
+* **Payment Premium (paymentPremiumPPB)** - <span>30</span>% gas premium
+    This premium is applied on top of the gas cost paid by the Keeper to perform your Upkeep
+* **Block Count Per Turn (blockCountPerTurn)** - <span>10</span>
+    The number of blocks in each turn before the Upkeep is rotated to the next Keeper
+* **Check Gas Limit (checkGasLimit)** - <span>6,500,000</span>
+    The maximum amount of gas that can be used by your checkUpkeep for off-chain computation.
+* **Call Gas Limit (callGasLimit)**: <span>2,500,000</span> <!-- @TODO render this programatically. <span id="show-callGasLimit"></span> -->
```

Comment:
for BSC callGasLimit is 5M, for all others is remains on 2.5M

Reply:
Fix applied.